src/core/biblio.js

```diff
@@ -20,6 +21,7 @@ const bibrefsURL = new URL("https://specref.herokuapp.com/bibrefs?refs=");
 /**
  * Normative references take precedence over informative ones,
  * so any duplicates ones are removed from the informative set.
+ * @param {Conf} conf
```

Comment:
I wonder we can make modules export a class instead, so that core/base-runner can create instances from modules like `new Biblio(conf)` and we can stop passing `conf` every time we call internal functions.

Just a thought for future refactoring.